flexibleengine/acceptance/resource_flexibleengine_sfs_turbo_test.go

```diff
@@ -67,6 +75,7 @@ func TestAccSFSTurbo_crypt(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "name", turboName),
 					resource.TestCheckResourceAttr(resourceName, "share_proto", "NFS"),
 					resource.TestCheckResourceAttr(resourceName, "share_type", "STANDARD"),
+					resource.TestCheckResourceAttr(resourceName, "enhanced", "false"),
```

Comment:
this parameter is missing in the docs

Reply:
it's not supported in FE, removed from the test